pkg/services/container/morph/executor.go

```diff
@@ -231,8 +231,16 @@ func (s *morphExecutor) SetExtendedACL(_ context.Context, tokV2 *sessionV2.Token
 		return nil, errors.New("missing signature")
 	}
 
+	var table eaclSDK.Table
+	eacl := body.GetEACL()
+	if eacl != nil {
+		if err := table.ReadFromV2(*body.GetEACL()); err != nil {
+			return nil, fmt.Errorf("can parse eacl body: %w", err)
```

Comment:
```suggestion
			return nil, fmt.Errorf("invalid eACL: %w", err)
```
we `can` not, and we decode whole eACL, not just `body`

Reply:
Replaced